### PR TITLE
New Box AI API default model

### PR DESCRIPTION
**Pull request by SowaProgramuje**
# Description

Please include a summary of the change and which issue is fixed. 

If you are a Boxer, please also reference the related `DDOC` or `APIWG` tickets.
If you do not have a related `Jira` ticket and this is more than a bug fix, then
create one now.

Fixes # (issue)
Re `DDOC-#` (`Jira` ticket)
Re `APIWG-#` (`Jira` ticket)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own changes
- [x] I have run `yarn test` and `yarn lint` to make sure my changes pass all
  linters and tests
- [x] I have pulled the latest changes from the upstream developer branch

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

**Review by shurrey — APPROVED** [review]

**Review by superojla — APPROVED** [review]
lgtm 🐸

File: content/2024/09-23-box-ai-api-defaul-model-update.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+is_impactful: true
+is_new_feature: true
+collapse: true
+show_excerpt: true
+release_source_url: ''
+---
+
+# Box AI API default model update
+
+Box AI Platform API has been updated with a new default model for `POST /2.0/ai/ask` endpoint. Both single and multiple documents modes are now based on Azure's `gpt-4o-mini`.
```
Comment (bszwarc) [inline content/2024/09-23-box-ai-api-defaul-model-update.md:15]:
```suggestion
Box AI Platform API has been updated with a new default model for the `POST /2.0/ai/ask` endpoint. Both single and multiple document modes are now based on Azure's `gpt-4o-mini`.
```

File: content/2024/09-23-box-ai-api-defaul-model-update.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+> **Note**: Box AI Platform API is currently a beta feature offered subject to Box’s Main Beta Agreement, and the available capabilities may change. Box AI Platform API is available to all Enterprise Plus customers.
+
+<!-- more -->
+
+For more information on [`default agent configuration`][1] and [`supported AI models`][2] visit developer guides.
```
Comment (bszwarc) [inline content/2024/09-23-box-ai-api-defaul-model-update.md:21]:
```suggestion
For more information on [default agent configuration][1] and [supported AI models][2] visit developer guides.
```

**Review by bszwarc — COMMENTED** [review]
Minor improvements only :)

**Pull request merged**